pkg/models/tenant/tenant.go

```diff
@@ -43,8 +46,29 @@ func CreateNamespace(workspaceName string, namespace *v1.Namespace, username str
 	}
 
 	namespace.Labels[constants.WorkspaceLabelKey] = workspaceName
+	created, err := client.ClientSets().K8s().Kubernetes().CoreV1().Namespaces().Create(namespace)
+	if err != nil {
+		klog.Error(err)
+		return nil, err
+	}
+
+	timeout := int64(10)
+	w, err := client.ClientSets().K8s().Kubernetes().RbacV1().RoleBindings(namespace.Name).Watch(metav1.ListOptions{TimeoutSeconds: &timeout})
```

Comment:
apiserver is only responsible for query, watching a object creation should be placed in controller